gateway/slack.go

```diff
@@ -144,16 +147,20 @@ type ClientChans struct {
 }
 
 // Initialize bootstraps the SlackClient with a client token and loads data
-func (sc *SlackClient) Initialize(token string) {
+func (sc *SlackClient) Initialize(token, userID string) (*SlackUser, error) {
```

Comment:
Now I'm not a go expert, but does `token` need a type?

Reply:
if you don't specify a type, it uses the next type to be specified. so `token, userID string` is the same as `token string, userID string` and so would `poop int, token, userID string`